build_tools/cmake/run_test.bat

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
I think this file can be removed, we don't support Windows

(also I think batch file has a different comment syntax)

Reply:
Done.